com.heroicgameslauncher.hgl.yml

```diff
@@ -0,0 +1,158 @@
+id: com.heroicgameslauncher.hgl
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+command: heroic-run
+separate-locales: false
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Compat.i386
+  - org.freedesktop.Sdk.Extension.toolchain-i386
+
+finish-args:
+  - --allow=devel
+  - --allow=multiarch
+  - --device=all
+  - --env=GST_PLUGIN_SYSTEM_PATH=/app/lib/gstreamer-1.0:/app/lib32/gstreamer-1.0:/usr/lib/extensions/gstreamer-1.0:/usr/lib/x86_64-linux-gnu/gstreamer-1.0:/usr/lib/i386-linux-gnu/gstreamer-1.0
```

Comment:
Unless you plan to bundle gstreamer modules, you can remove `/app/lib/gstreamer-1.0` and `/app/lib32/gstreamer-1.0` from here.

Reply:
We need that for proton to work.

Reply:
`/app` is where the HGL app bundle is mounted at run time. There are no gstreamer modules in it. You need the `/usr/.../gstreamer-1.0` paths, but not `/app/.../gstreamer-1.0` paths.

Reply:
Actually not, removed and proton still works :)